tests/user_commands/tests.py

```diff
@@ -535,3 +542,74 @@ def test_is_ignored_path_false(self):
     def test_normalize_path_patterns_truncates_wildcard_base(self):
         expected = [os.path.normcase(p) for p in ["foo/bar", "bar/*/"]]
         self.assertEqual(normalize_path_patterns(["foo/bar/*", "bar/*/"]), expected)
+
+    def test_get_directory_tree(self):
+        from django.utils.termcolors import colorize
+
+        def format_dir(x):
+            return colorize(x, fg="blue", opts=["bold"])
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            subdir = Path(tmpdir) / "foo/bar/"
+            subdir.mkdir(exist_ok=True, parents=True)
+            [
+                Path(tmpdir).joinpath("file1.txt").touch(),
+                Path(tmpdir).joinpath("file2.txt").touch(),
+                Path(tmpdir).joinpath("foo/foo1.txt").touch(),
+                Path(tmpdir).joinpath("foo/foo2.txt").touch(),
+                Path(tmpdir).joinpath("foo/bar/bar1.txt").touch(),
+                Path(tmpdir).joinpath("foo/bar/bar2.txt").touch(),
+            ]
+            expected = [
+                format_dir(Path(tmpdir).name),
+                f"{TEE_PREFIX}file1.txt",
+                f"{TEE_PREFIX}file2.txt",
+                f"{ELBOW_PREFIX}{format_dir('foo')}",
+                f"{SPACE_PREFIX}{TEE_PREFIX}{format_dir('bar')}",
+                f"{SPACE_PREFIX}{BRANCH_PREFIX}{TEE_PREFIX}bar1.txt",
+                f"{SPACE_PREFIX}{BRANCH_PREFIX}{ELBOW_PREFIX}bar2.txt",
+                f"{SPACE_PREFIX}{TEE_PREFIX}foo1.txt",
+                f"{SPACE_PREFIX}{ELBOW_PREFIX}foo2.txt",
```

Comment:
Please remove the constants, I think that would make this easier to read and see